samples/bluetooth/hci_spi/prj_nrf51_pca10028.conf

```diff
@@ -0,0 +1,28 @@
+CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_BLUETOOTH=y
+CONFIG_BLUETOOTH_HCI_RAW=y
+CONFIG_BLUETOOTH_MAX_CONN=20
+CONFIG_BLUETOOTH_TINYCRYPT_ECC=n
+CONFIG_BLUETOOTH_SPI_TO_HOST_DEV_NAME="SPI_1"
+CONFIG_BLUETOOTH_SPI_TO_HOST_IRQ_DEV_NAME="GPIO_0"
+CONFIG_BLUETOOTH_SPI_TO_HOST_IRQ_PIN=22
+CONFIG_BLUETOOTH_RX_BUF_LEN=255
+
+# Debug
+CONFIG_SYS_LOG=y
+CONFIG_SYS_LOG_DEFAULT_LEVEL=2
+CONFIG_BLUETOOTH_DEBUG_LOG=y
```

Comment:
with BLUETOOTH_DEBUG_LOG and SYS_LOG both enabled, I get compile warning: "SYS_LOG_LEVEL" redefined.

We dont need debug enabled in default config, please remove for these debug defines for both boards.